domain/consensus/processes/blockbuilder/test_block_builder.go

```diff
@@ -136,6 +136,11 @@ func (bb *testBlockBuilder) buildBlockWithParents(parentHashes []*externalapi.Do
 		return nil, nil, err
 	}
 
+	_, err = bb.difficultyManager.UpdateDAADataAndReturnDifficultyBits(tempBlockHash)
+	if err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
Why is this needed?

Reply:
This is a prerequisite for ResolveBlockStatus